pyproject.toml

```diff
@@ -12,7 +12,7 @@ cdiserrors = "~=1.0"
 xmltodict = "~=0.9"
 
 authlib = "~=0.11"
-httpx = "^0.12.1"
```

Comment:
can't poetry resolve `"^0.14.0" + "^0.12.1"` on its own?

Reply:
oh yeah it should. `^0.12.0` is basically `>=0.12.0` right?

Reply:
yeah, you shouldn't need this PR 🤔

Reply:
deleting this PR :+1: